taxonomies/ingredients.txt

```diff
@@ -10228,8 +10380,40 @@ fr:substances laitières modifiées
 
 
 <en:milk
-fr:lait fermenté
-nl:gefermenteerde melk
+en:soured milk
+bar:Gschdeglde
+bg:Кисело мляко
+ca:Llet agra
+cv:Турăх
+da:Tykmælk
+de:Dickmilch, Setzmilch, Stockmilch, Sauermilch
+es:Leche agria
+et:Hapupiim
+fr:lait fermenté, lait aigre, lait caillé
+is:Súrmjólk
+it:Latte acido
+ko:산유
+la:Oxygala
+lt:Rūgpienis
+mk:Кисело млеко
+nl:Dikmelk, gefermenteerde melk
+nn:Surmjølk
+no:Kulturmelk
+pl:Zsiadłe mleko
+pnb:لسی
+pt:Leite fermentado
+ro:Lapte acru
+ru:Простокваша
+sl:Kislo mleko
+so:Caano ciir
+sv:Surmjölk
+uk:Кисляк (молоко)
+zh_yue:酸奶
+zh:酸乳
+vegan:en:no
+vegetarian:en:yes
+wikipedia:en:https://en.wikipedia.org/wiki/Soured_milk
+wikidata:en:Q253333
 
 <en:milk
 fr:lait écrémé caillé
```

Comment:
I am a bit lost on the fermented stuff. Wasn't there a section with de:Buttermilch. If so maybe it should be a child of "soured milk?

Reply:
A nice overview over milk products in German can be found [here on wikipedia.org](https://de.wikipedia.org/wiki/Datei:Milch.svg).
```en:buttermilk``` is actually not just kind of soured milk. But we can add it as sub-category to the newly added category of ```en:fermented milk products```. There also other products could go, such as ```en:yogurt```, ```en:kefir```, ```en:soured milk```. @aleene Do you agree?

Reply:
Seems logic. Go ahead.
We had a discussion on fermented/sour a while back: https://en.wiki.openfoodfacts.org/Creams_-_en. Name/definition also depends on country and local regulation. So be careful with combining "translations".

Reply:
Done.